SECURITY.md

```diff
@@ -0,0 +1,98 @@
+
+### Fix Disclosure Process
+With the fix development underway, the Fix Lead needs to come up with an overall communication plan for the wider community. This Disclosure process should begin after the Fix Team has developed a Fix or mitigation so that a realistic timeline can be communicated to users.
+
+**Disclosure of Forthcoming Fix to Users** (Completed within 1-7 days of disclosure)
+- The Fix Lead will email kyverno@googlegroups.com informing users that a security vulnerability has been disclosed and that a fix will be made available at YYYY-MM-DD HH:MM UTC in the future via this list. This time is the Release Date.
+- The Fix Lead will include any mitigating steps users can take until a fix is available.
+
+The communication to users should be actionable. They should know when to block time to apply patches, understand exact mitigation steps, etc.
+
+**Fix Release Day** (Completed within 1-21 days of disclosure)
+- The maintainers will create a new patch release branch from the latest patch release tag + the fix from the security branch. As a practical example, if v1.5.3 is the latest patch release in kyverno.git, a new branch will be created called v1.5.4, which includes only patches required to fix the issue.
+- The Fix Lead will cherry-pick the patches onto the main branch and all relevant release branches. The Fix Team will LGTM and merge. Maintainers will merge these PRs as quickly as possible. Changes shouldn't be made to the commits, even for a typo in the CHANGELOG, as this will change the git SHA of the commits leading to confusion and potentially conflicts as the fix is cherry-picked around branches.
+- The Fix Lead will request a CVE from [DWF](https://github.com/distributedweaknessfiling/DWF-Documentation) and include the CVSS and release details.
+- The Fix Lead will email kyverno@googlegroups.com now that everything is public announcing the new releases, the CVE number, and the relevant merged PRs to get wide distribution and user action. As much as possible this email should be actionable and include links on how to apply the fix to user's environments; this can include links to external distributor documentation.
+- The Fix Lead will remove the Fix Team from the private security repo.
+
+### Retrospective
+These steps should be completed 1-3 days after the Release Date. The retrospective process [should be blameless](https://landing.google.com/sre/book/chapters/postmortem-culture.html).
+- The Fix Lead will send a retrospective of the process to kyverno@googlegroups.com, including details on everyone involved, the timeline of the process, links to relevant PRs that introduced the issue, if relevant, and any critiques of the response and release process.
+- Maintainers and Fix Team are also encouraged to send their own feedback on the process to kyverno@googlegroups.com. Honest critique is the only way we are going to get good at this as a community.
+
+
+### Embargo Policy
```

Comment:
Can you please share a reference for projects that use this.

Reply:
Yeah. Some of them are [Opa](https://github.com/open-policy-agent/opa/blob/main/SECURITY.md), [Envoy Proxy](https://github.com/envoyproxy/envoy/blob/main/SECURITY.md). They mainly use it for their private distributors.